### PR TITLE
DCA-1799 add action time to jmeter logs

### PR DESCRIPTION
**Pull request by mbrzo2**

### CODE REVIEW

File: app/jmeter/jira.jmx [hunk 1]
```diff
@@ -2964,8 +2966,11 @@ def delay_between_transactions = (1000 * 3600) / (vars.get(&quot;total_actions_p
 //Sets the pacing length based on the last requests response time. x is the time in ms
 def start_time = Long.valueOf(vars.get(&quot;loop_start_time&quot;))
 def sleep_time = (delay_between_transactions - (System.currentTimeMillis() - start_time)) as int
+def sleep_time = (delay_between_transactions - (System.currentTimeMillis() - start_time)) as int
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:2969]: Looks like a duplicate line.

File: app/jmeter/jira.jmx [hunk 2]
```diff
@@ -3498,8 +3503,11 @@ def delay_between_transactions = (1000 * 3600) / (vars.get(&quot;total_actions_p
...
```
Comment (ometelytsia) [inline app/jmeter/jira.jmx:3506]: Duplicate linet.

File: app/jmeter/jsm_agents.jmx [hunk 3]
```diff
@@ -7748,8 +7752,10 @@ def delay_between_transactions = (1000 * 3600) / (vars.get(&quot;total_actions_p
 //Sets the pacing length based on the last requests response time. x is the time in ms
 def start_time = Long.valueOf(vars.get(&quot;loop_start_time&quot;))
 def sleep_time = (delay_between_transactions - (System.currentTimeMillis() - start_time)) as int
+def sleep_time = (delay_between_transactions - (System.currentTimeMillis() - start_time)) as int
```
Comment (ometelytsia) [inline app/jmeter/jsm_agents.jmx:7755]: Duplicate line.

Comment (mbrzo2) [reply app/jmeter/jsm_agents.jmx:7755]: fixed

**Review by ometelytsia — APPROVED** [review]

**Pull request merged**